nnpdf_data/nnpdf_data/utils.py

```diff
@@ -17,14 +12,14 @@ def parse_yaml_inp(input_yaml, spec):
     https://validobj.readthedocs.io/en/latest/examples.html#yaml-line-numbers
     """
     input_yaml = pathlib.Path(input_yaml)
-    inp = yaml.load(input_yaml.read_text(encoding="utf-8"), Loader=Loader)
+    inp = yaml.load(input_yaml.read_text(encoding="utf-8"), yaml.Loader)
     try:
         return parse_input(inp, spec)
     except ValidationError as e:
         current_exc = e
         # In order to provide a more complete error information, use round_trip_load
         # to read the .yaml file again (insetad of using the CLoader)
-        current_inp = yaml.round_trip_load(input_yaml.open("r", encoding="utf-8"))
+        current_inp = yaml.load(input_yaml.open("r", encoding="utf-8"), yaml.Loader)
```

Comment:
Turns out ruamel is built on top of pyyaml, so CLoader is the same thing in both. I took the passing ci as indication that this was fine, but I'll revert it

Reply:
Then you need to use CLoader for the other one to speed things up (and you need the try-except since libyaml might not be installed everywhere).
I think all in all it is better to use ruamel also here, pyyaml doesn't really simplify things.

> I took the passing ci as indication that this was fine, but I'll revert it

We would need a test that not only checks for errors but looks also at the message... and in this case is a `ValidationError` from reportengine. I'll put it in my "probably never happening but who knows" to do list